scripts/docker/dev-valgrind/Dockerfile

```diff
@@ -0,0 +1,92 @@
+# vim:set ft=dockerfile:
```

Comment:
Should these dockerfiles be in a separate repo? I think they are convenient to have here but not opposed to another place

Reply:
In my opinion, yes. We previously moved a lot of stuff like this out of the main repo to keep it clean and this risks cluttering things up again. Besides, it seems building these test-images is a one time thing that is orthogonal to main development of the DB. It might even be something that other people doing PostgreSQL development are interested in so warrants its own repo for sure.